<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="info_tab">
       <attribute name="title">
        <string>Информация о системе</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="host_name_label">
            <property name="text">
             <string>Имя компьютера:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="user_name_label">
            <property name="text">
             <string>Имя пользователя:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="uptime_label">
            <property name="text">
             <string>Время непрерывной работы:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Модель центрального процессора:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="cpu_frequency_label">
            <property name="text">
             <string>Частота центрального процессора:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="cpu_load_label">
              <property name="text">
               <string>Нагрузка процессора:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="cpu_load_progressbar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="total_memory_label">
            <property name="text">
             <string>Доступная оперативная память:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="busy_memory_lable">
              <property name="text">
               <string>Используемая оперативная память:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="busy_memory_progressbar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="process_table_tab">
       <attribute name="title">
        <string>Процессы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="process_table_tablewidget"/>
          </item>
          <item>
           <widget class="QPushButton" name="kill_process_pushbutton">
            <property name="text">
             <string>Завершить процесс</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
